packages/@sanity/form-builder/src/inputs/PortableText/Editor.styles.tsx

```diff
@@ -64,6 +67,7 @@ export const EditableWrapper = styled(Card)<{$isFullscreen: boolean; $readOnly:
   width: 100%;
   counter-reset: ${TEXT_LEVELS.map((l) => createListName(l)).join(' ')};
   overflow: hidden;
+  --card-bg-color: 'inherit';
```

Comment:
It is invalid to use quotes here, should be:

`--card-bg-color: inherit`

Reply:
Thank you!